plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -110,26 +132,31 @@ const renderTabs = ( contextValue: MarketplaceContextType ) => {
 const Tabs = ( props: TabsProps ): JSX.Element => {
 	const { additionalClassNames } = props;
 	const marketplaceContextValue = useContext( MarketplaceContext );
-	const { setSelectedTab } = marketplaceContextValue;
+	const { selectedTab, setSelectedTab } = marketplaceContextValue;
+	const [ activeTabs, setActiveTabs ] = useState( getActiveTabs( '' ) );
```

Comment:
When you say active, I think you mean visible, right? For a moment I confused active with selected.